monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -103,16 +104,17 @@ import scala.util.{Failure, Success, Try}
   * has real consequences. For example with `Task` you can do this:
   *
   * {{{
-  *   def retryOnFailure[A](times: Int, source: Task[A]): Task[A] =
-  *     source.onErrorRecoverWith { err =>
-  *       // No more retries left? Re-throw error:
-  *       if (times <= 0) Task.raiseError(err) else {
-  *         // Recursive call, yes we can!
-  *         retryOnFailure(times - 1, source)
-  *           // Adding 500 ms delay for good measure
-  *           .delayExecution(500)
-  *       }
-  *     }
+  *   scala> import scala.concurrent.duration._
+  *   scala> def retryOnFailure[A](times: Int, source: Task[A]): Task[A] =
+  *        | source.onErrorHandleWith { err =>
```

Comment:
here `onErrorRecoverWith` is `replaced with `onErrorHandleWith` as it's total and `onErrorRecoverWith` is partial